demo/body3d_two_stage_video_demo.py

```diff
@@ -145,6 +153,36 @@ def main():
         default='configs/_base_/filters/one_euro.py',
         help='Config file of the filter to smooth the pose estimation '
         'results. See also --smooth.')
+    parser.add_argument(
+        '--return-heatmap',
+        action='store_true',
+        default=False,
+        help='whether to return heatmap')
+    parser.add_argument(
+        '--output-layer-names',
+        nargs='*',
+        type=str,
+        help='return the output of some desired layers, '
+        'e.g. use ("backbone", ) to return backbone feature')
+
+    parser.add_argument(
+        '--use-multi-frames',
+        action='store_true',
+        default=False,
+        help='whether to use multi frames for inference. Default: False.')
+    parser.add_argument(
+        '--online',
+        action='store_true',
+        default=False,
+        help='inference mode. If set to True, can not use future frame'
+        'information when using multi frames for inference. Default: False.')
+    parser.add_argument(
+        '--save-memory',
+        action='store_true',
+        default=False,
+        help='save memory usage when using large model for inference. If you'
+        'have plenty of memory, you can turn it off to gain faster inference'
+        'speed. Default: True.')
```

Comment:
Is this a specific argument for PoseWarper? If so, we better not add an argument (especially with such a general argument name) in this demo script. It can be set in the config of the PoseWarper model. For example, you can add a config file for inference in memory saving mode.

Reply:
For now, it is a specific argument for PoseWarper. Yes, it can be set in the config of the PoseWarper model. Because it only needs some minor changes, I don't think it is worth adding a new config file for inference.

Reply:
BTW, how about adding an argument `--cfg-options` like in `tools/train.py`, so that we can change the config option without adding a new config file?